src/amoc_throttle/amoc_throttle_process.erl

```diff
@@ -117,11 +116,13 @@ handle_continue(maybe_run_fn, State) ->
     NewState = maybe_run_fn(State),
     {noreply, NewState, timeout(NewState)}.
 
+-spec format_status(term(), term()) -> term().
```

Comment:
Is it anyhow better than automatically generated spec?

Reply:
is it possible to exclude that function from ex_doc, and the whole file as well?  it is an internal module.